data/product.d/fedora-iot.conf

```diff
@@ -0,0 +1,11 @@
+# Anaconda configuration file for Fedora IoT.
+
+[Product]
+product_name = Fedora
```

Comment:
I have tested it and the product name is supposed to be `Fedora-IoT`. The variant name is correct.

Reply:
Something is not right here, because as far as I can tell, no other Edition installed through Anaconda is configured that way. Silverblue, Workstation, and old Fedora Atomic Host were just `Fedora` product with the Edition name as the variant. Why is this different?

Reply:
Anaconda reads the product and variant names from the `.buildstamp` file that is generated by `lorax`. I guess that `lorax` is run with arguments `--product "Fedora-IoT" --variant "IoT"`. I don't know where this configuration comes from.

Reply:
That sounds like a bug that needs to be fixed.

Reply:
I think it's [Basically this](https://pagure.io/fedora-iot/issue/24) - I think lorax uses the productmd 'short name' here. We might actually want to allow some kind of wildcard specification here, as we may well wind up with IoT images with `product_name` "Fedora" at some point, and there are other cases where we have images for the same variant produced with different `product_name`s, e.g. the [post-release nightly Cloud composes](https://kojipkgs.fedoraproject.org/compose/cloud/) will likely use "Fedora-Cloud", if I'm understanding correctly here.

Reply:
I can't help myself but I see this as logically bad decision. It's still product Fedora not Fedora-IoT. For me it feels that even when they changed how they are making composes it should not touch the product name.

Reply:
I have replied to the pagure issue.

Reply:
Well, there are practical reasons we need more-or-less that value to differ between different compose streams. We can't have two "Fedora" compose streams because it'd produce all kinds of ambiguities - the dist name is expected to be a unique identifier in, for instance, the compose ID (we'd often wind up with two or more entirely different composes with the same ID) and many deliverable file names (if two compose streams with some difference between them both build an image with the same subvariant, for instance, and we run both streams on the same day, they'd likely produce two different but identically-named image files).

At the Pungi and productmd levels this property is not referred to as the "product" name, FWIW. It's the "release" "short" name. It sort of 'becomes' the "product" name at the lorax level.

Reply:
Thanks a lot for explanation.

In that case there is a question if these shouldn't be separate? I mean if Pungi should not put something else to Lorax to avoid this problem.

Reply:
I mean, that's something we can look at, but it's still the sort of thing I'd worry will unexpectedly break something else. We're fairly clear at least on how the current behaviour works. Can we not just teach anaconda wildcards for this or something, so we can do product configs with 'Fedora*'?

Reply:
@jkonecny12 @AdamWill How about I make a separate commit that just changes the product to `Fedora-IoT` that can be reverted as soon as this mess is fixed in the compose tooling?

Reply:
@poncovka @jkonecny12 @AdamWill I've added a commit to mangle the name until this is fixed.